image-layout.md

```diff
@@ -15,9 +15,12 @@ The image layout has two top level directories:
 - "refs" contains [descriptors][descriptors]. Commonly pointing to an [image manifest](manifest.md#image-manifest) or an [image manifest list](manifest-list.md#oci-image-manifest-list-specification).
 
 
-It also contains a file that is used to identify the layout version:
+The image layout MUST contain a file "oci-layout":
 
-- "oci-layout" MUST contain a JSON object with a version field `{"imageLayoutVersion": "1.0.0"}` and MAY include additional fields.
+- It MUST be a JSON object
+- It MUST contain a `imageLayoutVersion` field for determing the version of the layout used
+    - The `imageLayoutVersion` value will align with the `specs.Version` at the time changes to the layout are made, and will pin a given version until changes to the layout are required
```

Comment:
It may be worth pointing out that this means `imageLayoutVersion` is not semantically versioned.  I'm ok with that, since I expect most image-layout bumps will break backwards compat anyway, but my personal preference would be to have a semantically versioned `imageLayoutVersion` that is more independent of the spec version.  You'd bump the spec version if *any* constituent component added a new feature (minor bump) or a backwards-incompatible feature (major bump).  So:

* A major `imageLayoutVersion` bump would require a major spec version bump.
* A minor `imageLayoutVersion` bump would require at least a minor spec version bump.
* A new layer media type (e.g. for #24) would require a minor spec version bump but no `imageLayoutVersion` bump.
* Dropping an old media type (e.g. after types based on `.wh.*` fall out of use) would require a major spec version bump but no `imageLayoutVersion` bump.